lib/make.js

```diff
@@ -47,7 +47,12 @@ const cache = path.resolve(cacheRoot, cacheExtra);
 let defaults;
 let globalPrefix;
 
-${body.replace(INDENT_REGEXP, INDENT_REPLACER).replace("'node-version': process.version,", '// We remove node-version to fix the issue described here: https://github.com/pnpm/pnpm/issues/4203#issuecomment-1133872769')};
+${body
+	.replace(INDENT_REGEXP, INDENT_REPLACER)
+	.replace("'node-version': process.version,", '// We remove node-version to fix the issue described here: https://github.com/pnpm/pnpm/issues/4203#issuecomment-1133872769')
+	 // add provenance: https://github.com/pnpm/pnpm/pull/6436
+	.replace("proxy: null,", "proxy: null,\n\t\t\tprovenance: false,")
```

Comment:
I don't understand. Isn't this package fetching code from npm CLI? So why isn't this setting in the npm settings?

Reply:
Then we either need to update the location or disable code generation.

Reply:
maybe we need disable code generation,I will update it later~